ui/apps/platform/src/Containers/SystemHealth/Components/BackupIntegrationHealthWidget.tsx

```diff
@@ -5,14 +5,15 @@ import { fetchBackupIntegrations } from 'services/BackupIntegrationsService';
 import integrationsList from 'Containers/Integrations/utils/integrationsList';
 import IntegrationHealthWidgetVisual from './IntegrationHealthWidgetVisual';
 import { mergeIntegrationResponses, IntegrationMergedItem } from '../utils/integrations';
+import { getAxiosErrorMessage } from '../../../utils/responseErrorUtils';
```

Comment:
Although not a blocker to merge, `'utils/responseErrorUtils` is more descriptive for import from utils folder.